coalib/collecting/Collectors.py

```diff
@@ -170,7 +170,9 @@ def collect_bears(bear_dirs, bear_globs, kinds, log_printer,
 
     if warn_if_unused_glob:
         _warn_if_unused_glob(log_printer, bear_globs, bear_globs_with_bears,
-                             "No bears were found matching '{}'.")
+                             "No bears were found matching '{}'. Make sure you "
+                             "typed the name correctly or install it from here "
+                             "https://github.com/coala-analyzer/coala-bears.")
```

Comment:
uhm, the user would have installed it from pip or git repo already, so this message definitely doesn't help. It's rather wrong